### DISCUSSION TITLE
OpenAI now allows developers to customise GPT-3 models

### DISCUSSION THREAD

**Discussion by jacky0115** [1101-GS3073-C/Python-Community-GS3073-C#188] [科技新知]
[OpenAI now allows developers to customise GPT-3 models](https://artificialintelligence-news.com/2021/12/15/openai-now-allows-developers-to-customise-gpt-3-models/)
OpenAI現在允許開發者可以對GPT-3進行finetuning，這樣就等於是踩在巨人的肩膀上、直接在1750億參數的大模型上微調，不少研發人員都可以不用再重頭訓練自己的AI模型了，finetuning相比few-shot-learning通常能花費更少的成本就能得到高速且品質好的訓練表現，OpenAI此一決策無非就是對AI開發者們的一大福音阿

**Comment by briankuo123** [comment]
聽起來真是個好消息，光是期末專題的AI要去做簡單的訓練我就快受不了了。

**Comment by wh20502** [comment]
以前寫程式都是要寫最低階的從CMD那種，到現在有很多都幫忙包裝好的高階程式，我想未來AI程式設計一定也會有愈來越多高階的程式，讓大家更好上手。
